rp2040-hal/src/rosc.rs

```diff
@@ -1,5 +1,16 @@
 //! Ring Oscillator (ROSC)
-// See [Chapter 2 Section 17](https://datasheets.raspberrypi.org/rp2040/rp2040_datasheet.pdf) for more details
+//! See [Chapter 2 Section 17](https://datasheets.raspberrypi.org/rp2040/rp2040_datasheet.pdf) for more details
+//!
```

Comment:
Add a blank line here so the "See chapter" comment doesn't become part of the heading (which is copied into the parent document)
```suggestion
//!
//! See [Chapter 2 Section 17](https://datasheets.raspberrypi.org/rp2040/rp2040_datasheet.pdf) for more details
//!
```